core/remote/cozy.js

```diff
@@ -274,3 +273,19 @@ module.exports = {
   FSCK_PATH,
   RemoteCozy
 }
+
+async function getChangesFeed (since, client) {
+  const response = await client.data.changesFeed(FILES_DOCTYPE, { since, include_docs: true, limit: 10000 })
+  const {last_seq, pending, results} = response
+  if (pending === 0) {
+    return response
+  }
+  const nextResponse = await getChangesFeed(last_seq, client)
+  return {
+    ...nextResponse,
```

Comment:
Can object spread/rest be used in node 8? I think it is but I'd like to have a confirmation

Reply:
O_o 
```
➜  cozy-desktop (fix/changesfeedtimeout) ✗ yarn start
yarn run v1.12.1
$ env-cmd .env.dev electron .
App threw an error during load
cozy/cozy-desktop/core/remote/cozy.js:285
    ...nextResponse,
    ^^^

SyntaxError: Unexpected token ...
```

Reply:
fixed!

Reply:
I believe it's available [in 8.9.4 but not in 8.2.1](https://node.green/#ES2018-features-object-rest-spread-properties-object-rest-properties) (which is the one available with our current electron version) without a flag.